### PR TITLE
Adding ZMO-606-P2, ZMO-606-S2 and ZMO-606-20A to tuya.js

### PR DESCRIPTION
**Pull request by fabergoncalves**
Adding to tuya.js:

- ZMO-606-P2: 2 power outlets,
- ZMO-606-S2: 2 switches and 1 power outlet
- ZMO-606-20A: 1 20A power outlet

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -8845,6 +8845,53 @@ const definitions: Definition[] = [
...
+        ],
+        model: 'ZMO-606-S2',
+        vendor: 'Zemismart',
+        description: 'Zemismart Smart 2 Gangs Switch with Outlet',
+        extend: [
+            deviceEndpoints({'endpoints': {'1': 1, '2': 2, '3': 3}}),
+            identify(),
+            tuya.modernExtend.tuyaOnOff({indicatorMode: true, onOffCountdown: true, endpoints: ['1', '2', '3']}),
+        ],
+        meta: {'multiEndpoint': true},
```
Comment (Koenkk) [inline src/devices/tuya.ts:8880]: This is not needed, `meta` can be removed for all

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -8845,6 +8845,53 @@ const definitions: Definition[] = [
...
+            {
+                modelID: 'TS0003',
+                manufacturerName: '_TZ3000_aknpkt02',
+            },
+        ],
+        model: 'ZMO-606-S2',
+        vendor: 'Zemismart',
+        description: 'Zemismart Smart 2 Gangs Switch with Outlet',
+        extend: [
+            deviceEndpoints({'endpoints': {'1': 1, '2': 2, '3': 3}}),
```
Comment (Koenkk) [inline src/devices/tuya.ts:8876]:
```suggestion
            deviceEndpoints({'endpoints': {'l1': 1, 'l2': 2, 'l3': 3}}),
```

And update `endpoints: ['l1', 'l2', 'l3']`

Make the same change for the `ZMO-606-P2`

File: src/devices/tuya.ts [hunk 3]
```diff
@@ -8845,6 +8845,53 @@ const definitions: Definition[] = [
             }
         },
     },
+    {
```
Comment (Koenkk) [inline src/devices/tuya.ts:8848]: Please move all to `zemismart.ts`

File: src/devices/tuya.ts [hunk 4]
```diff
@@ -8845,6 +8845,53 @@ const definitions: Definition[] = [
...
+        fingerprint: [
```
Comment (Koenkk) [inline src/devices/tuya.ts:8849]: Use `tuya.fingerprint` for all (search in this file for examples)

File: src/devices/tuya.ts [hunk 5]
```diff
@@ -8845,6 +8845,53 @@ const definitions: Definition[] = [
...
+            {
+                modelID: 'TS011F',
+                manufacturerName: '_TZ3000_b1q8kwmh',
+            },
+        ],
+        model: 'ZMO-606-20A',
+        vendor: 'Zemismart',
+        description: 'Zemismart Smart 20A Outlet',
```
Comment (Koenkk) [inline src/devices/tuya.ts:8891]:
```suggestion
        description: 'Smart 20A outlet',
```

For all: remove vendor name and make lower case

**Comment by fabergoncalves** [comment]
All suggestions were implemented

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**